docs/api/store-api.md

```diff
@@ -6,6 +6,25 @@ Apicart Store API listen at [https://store-api.apicart.net](https://store-api.ap
 Full GraphQL schema for Store API is available [here](https://docs.store-api.apicart.net).
 :::
 
+
+## Item files
+
+Mutation handling items in cart (e.g. `addCartItems` or `removeCartItems`) getting data from item files. These files describe the item and define, for example, its name or price.
```

Comment:
Mutations manipulating with products .... are getting data from product files. These files describes... and defines for example its name or price.